.github/workflows/pr-to-discord.yml

```
@@ -0,0 +1,41 @@
+                    "inline": true
+                  },
+                  {
+                    "name": "👤 작성자",
+                    "value": "${{ github.event.pull_request.user.login }}",
+                    "inline": true
+                  }
+                ]
+              }
+            ]
+          }
+EOF
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 41: could not find expected ':'

(syntax-check)